+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Fri Apr 28 09:08:42 2023 - INFO  - Start environment preparation.
+ DNF_INSTALL nfs-utils
+ pkgs=nfs-utils
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs nfs-utils --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpgs7ifqtj
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpgs7ifqtj
+ systemctl stop firewalld
Failed to stop firewalld.service: Unit firewalld.service not loaded.
+ iptables -F
+ SSH_CMD 'systemctl stop firewalld;iptables -F' 10.198.114.3 'openEuler12#$' root
+ cmd='systemctl stop firewalld;iptables -F'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'systemctl stop firewalld;iptables -F' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 09:09:58 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 09:09:59 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ LOG_INFO 'Environmental preparation is over.'
+ message='Environmental preparation is over.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Environmental preparation is over.'
Fri Apr 28 09:09:59 2023 - INFO  - Environmental preparation is over.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase.'
+ message='Start executing testcase.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase.'
Fri Apr 28 09:10:00 2023 - INFO  - Start executing testcase.
+ SSH_CMD 'yum install nfs-utils -y;mkdir /home/nfs;chmod 777 /home/nfs;mv /etc/exports /etc/exports.bak;
	echo '\''/home/nfs *(rw,sync,all_squash)'\'' >/etc/exports' 10.198.114.3 'openEuler12#$' root
+ cmd='yum install nfs-utils -y;mkdir /home/nfs;chmod 777 /home/nfs;mv /etc/exports /etc/exports.bak;
	echo '\''/home/nfs *(rw,sync,all_squash)'\'' >/etc/exports'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'yum install nfs-utils -y;mkdir /home/nfs;chmod 777 /home/nfs;mv /etc/exports /etc/exports.bak;
	echo '\''/home/nfs *(rw,sync,all_squash)'\'' >/etc/exports' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
Fri Apr 28 09:10:01 2023 - WARN  - the remote user uses the default configuration.
Fri Apr 28 09:10:02 2023 - WARN  - the remote password uses the default configuration.
Fri Apr 28 09:10:03 2023 - WARN  - the connect port using the default configuration
spawn ssh -o ConnectTimeout=300 -p 22 root@10.198.114.3 yum install nfs-utils -y;mkdir /home/nfs;chmod 777 /home/nfs;mv /etc/exports /etc/exports.bak;
	echo '/home/nfs *(rw,sync,all_squash)' >/etc/exports
root@10.198.114.3's password: 
Last metadata expiration check: 1:18:42 ago on Fri 28 Apr 2023 07:51:39 AM CST.
Package nfs-utils-2:2.6.2-3.oe2303.riscv64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!
mkdir: cannot create directory ‘/home/nfs’: File exists
+ ret=0
+ test 0 -ne 0
+ return 0
+ SSH_CMD 'exportfs -avr' 10.198.114.3 'openEuler12#$' root
+ cmd='exportfs -avr'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'exportfs -avr' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 09:10:33 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 09:10:34 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ CHECK_RESULT 101
+ actual_result=101
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 101 ']'
+ '[' 0 -eq 0 ']'
+ test 101x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_nfs_share_mount.sh line 35'
+ message='oe_test_storage_nfs_share_mount.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_nfs_share_mount.sh line 35'
Fri Apr 28 09:10:35 2023 - ERROR - oe_test_storage_nfs_share_mount.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD 'systemctl restart nfs-server rpcbind' 10.198.114.3 'openEuler12#$' root
+ cmd='systemctl restart nfs-server rpcbind'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'systemctl restart nfs-server rpcbind' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
1 packets transmitted, 0 received, 100% packet loss, time 0ms
Fri Apr 28 09:10:46 2023 - ERROR - connection to 10.198.114.3 failed.
+ ret=101
+ test 101 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:101'
+ message='Failed in remote CMD operation:101'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:101'
Fri Apr 28 09:10:47 2023 - ERROR - Failed in remote CMD operation:101
+ return 101
+ mkdir /home/client
+ systemctl restart nfs-server rpcbind
+ mount -t nfs 10.198.114.3:/home/nfs /home/client
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ df -h
+ grep 10.198.114.3
10.198.114.3:/home/nfs   20G  5.8G   13G  32% /home/client
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End of testcase execution.'
+ message='End of testcase execution.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of testcase execution.'
Fri Apr 28 09:11:56 2023 - INFO  - End of testcase execution.
+ CASE_RESULT 0
+ case_re=0
+ test -z 1
+ test 1 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Apr 28 09:11:56 2023 - ERROR - failed to execute the case.
+ exit 1
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Apr 28 09:11:57 2023 - INFO  - start environment cleanup.
+ umount /home/client
+ rmdir /home/client
+ SSH_CMD 'rm -rf /home/nfs;yum remove -y nfs-utils;mv -f /etc/exports.bak /etc/exports' 10.198.114.3 'openEuler12#$' root
+ cmd='rm -rf /home/nfs;yum remove -y nfs-utils;mv -f /etc/exports.bak /etc/exports'
+ remoteip=10.198.114.3
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'rm -rf /home/nfs;yum remove -y nfs-utils;mv -f /etc/exports.bak /etc/exports' -i 10.198.114.3 -u root -p 'openEuler12#$' -t 300 -o 22
Fri Apr 28 09:13:30 2023 - WARN  - the remote user uses the default configuration.
Fri Apr 28 09:13:30 2023 - WARN  - the remote password uses the default configuration.
Fri Apr 28 09:13:31 2023 - WARN  - the connect port using the default configuration
spawn ssh -o ConnectTimeout=300 -p 22 root@10.198.114.3 rm -rf /home/nfs;yum remove -y nfs-utils;mv -f /etc/exports.bak /etc/exports
ssh: connect to host 10.198.114.3 port 22: Connection timed out
+ ret=255
+ test 255 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:255'
+ message='Failed in remote CMD operation:255'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:255'
Fri Apr 28 09:15:42 2023 - ERROR - Failed in remote CMD operation:255
+ return 255
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpgs7ifqtj ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpgs7ifqtj
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup.'
+ message='Finish environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup.'
Fri Apr 28 09:16:12 2023 - INFO  - Finish environment cleanup.
